.gitignore

```diff
@@ -1,2 +1,2 @@
-.idea/
-wasm/target
+/target
+/Cargo.lock
```

Comment:
I think we should follow the convention of having trailing newlines in text files